Cue/app/common/SyncConflict.js

```diff
@@ -62,38 +120,107 @@ class SyncConflict extends React.Component {
   }
 
   _getRightItems = () => {
-    if (this.state.conflicts && !this.state.conflicts.find(c => c.useServerDeck==undefined)) {
+    if (this.state.conflicts && !this.state.conflicts.find(c => c.useServerDeck==null)) {
       return [{
         title: 'Done',
         icon: CueIcons.done,
         onPress: () => {
           let promises = []
           this.state.conflicts.forEach(conflict => {
-            promises.push(this.props.resolveConflict(conflict.useServerDeck, conflict.localDeck))
+            promises.push(this.props.resolveConflict(conflict))
           })
           Promise.all(promises)
           .then(this.props.navigator.pop())
           .catch(e => {
-            console.error(e)
-            Alert.alert ('Alert', 'Failed to resolve conflicts')
+            this._internetAlert()
             this.props.navigator.pop()
           })
         }
       }]
     }
   }
 
-  _setSelection = (deck: {}, useServerDeck: boolean) =>{
+  _setSelection = (uuid: string, useServerDeck: boolean) =>{
+    let conflicts = this.state.conflicts.map(conflict=> {
+      if (conflict.change.uuid == uuid)
+        return {...conflict, useServerDeck}
+      else
+        return conflict
+    })
+    let ds = new ListView.DataSource({
+      rowHasChanged: (r1, r2) => r1 != r2
+    })
     this.setState({
-      conflicts: this.state.conflicts.map(conflict => {
-        if (conflict.localDeck.uuid == deck.uuid)
-          return {...conflict, useServerDeck}
-        else
-          return conflict
-      })
+      dataSource: ds.cloneWithRows(conflicts),
+      conflicts,
     })
   }
 
+  _timeSince(date: Date) {
+    let seconds = Math.floor((new Date() - date) / 1000);
+    let interval = Math.floor(seconds / 31536000);
+
+    if (interval > 1) {
+      return interval + " years";
+    }
+    interval = Math.floor(seconds / 2592000);
+    if (interval > 1) {
+      return interval + " months";
+    }
+    interval = Math.floor(seconds / 86400);
+    if (interval > 1) {
+      return interval + " days";
+    }
+    interval = Math.floor(seconds / 3600);
+    if (interval > 1) {
+      return interval + " hours";
+    }
+    interval = Math.floor(seconds / 60);
+    if (interval > 1) {
+      return interval + " minutes";
+    }
+    return Math.floor(seconds) + " seconds";
+  }
+
+  _renderRow = (conflict) => {
+    let serverText
+    if (conflict.serverDeck) {
+      serverText = conflict.serverDeck.last_update_device
+        ? 'Keep changes from "' +  conflict.serverDeck.last_update_device + '"'
+        : 'Keep server copy'
+    } else {
+      serverText = 'Keep deleted server change'
```

Comment:
Maybe "Delete this deck" and make the subtext "Deleted on another device" or something?